src/Components/App/App.js

```diff
@@ -21,12 +22,11 @@ export default class App extends Component {
   }
 
   componentDidMount() {
-    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-      .then(response => response.json())
+      getMovies()
       .then(data => {
-        this.setState({movies: data.movies})
+        this.setState({ movies: data.movies })
       })
-      .catch(error => console.log(error))
+      .catch(error => this.setState({ error }))
```

Comment:
After our discussion, we'll want to refactor this to handle the 500. We can simply let the user know something went wrong!

Reply:
I just pushed up the update for this error handling - thanks T!

Reply:
Thank you for the quick fix!